apigentools/cli.py

```diff
@@ -72,10 +82,33 @@ def cli(ctx, **kwargs):
     ctx.obj = dict(kwargs)
     toplog = logging.getLogger(__name__.split(".")[0])
     set_log(toplog)
+    check_min_version(ctx)
```

Comment:
can we add an option `--skip-version-check`?

Reply:
Yeah we can add that. My thinking was that you'd implicitly skip if the field wasn't defined in the spec yaml file. But I can see this being useful for users that don't have control over the spec files and need to skip.